apps/common/fields/index.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Does the app still build if we remove this file?

Reply:
It is necessary as it is not building